templates/_linux.yaml

```diff
@@ -114,4 +114,9 @@ parameters:
 - name: PVCNAME
   description: Name of the PVC with the disk image
   required: true
-
+- name: SRCPVCNAME
+  description: Name of the PVC to clone
+  value: '{{ oslabels | last }}'
```

Comment:
What values would occur here? Can you give a few examples?

Reply:
e.g.
```
rhel8.2
rhel7.9
centos8
```

Reply:
Are all of these valid PVC names?
I suspect that anyething with a dot is not.

Reply:
Names with dot are valid k8s names